browser/ui/webui/webcompat_reporter/webcompat_reporter_ui.h

```diff
@@ -63,7 +63,8 @@ class WebcompatReporterDOMHandler : public content::WebUIMessageHandler {
 
   raw_ptr<content::RenderWidgetHostView> render_widget_host_view_;
   scoped_refptr<base::SequencedTaskRunner> ui_task_runner_;
-  std::unique_ptr<webcompat_reporter::WebcompatReportUploader> uploader_;
+  std::unique_ptr<webcompat_reporter::WebcompatReporterService>
+      reporter_service_;
```

Comment:
is this variable used somehow or is it a leftover? It looks wrong, because you should never own a `KeyedService`.

Reply:
it was leftover